docs/chapters/data-model/entities.md

```diff
@@ -345,4 +345,46 @@ And the schema resulting from it:
 }
 ```
 
-Now you can easily implement a middleware that, reading that schema, increment all the fields containing the `addDays` metadata by `n` days before insertion.
\ No newline at end of file
+Now you can easily implement a middleware that, reading that schema, increment all the fields containing the `addDays` metadata by `n` days before insertion.
+
+Here's another example of a field with metadata that has more than one key.
+
+```typescript
+type SomeType @entity @mongodb {
+  id: ID! @id(from: "db") @alias(value: "_id")
+  someField: Date @schema(metadata: [{ key: "keyOne", value: true }, { key: "keyTwo", value: [ "one", 2, "three", false ]}])
```

Comment:
Are you sure that a different type from String can be specified for the `value`?
The GraphQL definition is 
```typescript
input KeyValue { key: String!, value: String! }
directive @schema(metadata: [KeyValue!]!) on FIELD_DEFINITION
```

Im not sure on what happen if something different from String is defined. Have you tryed?

See https://github.com/twinlogix/typetta/blob/53a22eb5d49184eb91d8a27f007df7c364a54e46/src/utils/utils.typeDefs.ts#L21

Reply:
Yea, I did some testing locally to figure out what was supported and what was not.